src/app/AccountOne.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+
+export default function AccountOne() {
+  return (
+    <div className="account">
+      <h1>Account One</h1>
+      <NavLink to="/">Back to home</NavLink>
+    </div>
+  );
```

Comment:
you can create one generic component for accounts, where you can pass `Account One` text with help to props